src/plugins/simulator/components/simulator-tile.tsx

```diff
@@ -0,0 +1,61 @@
+import { VariableType } from "@concord-consortium/diagram-view";
+import { observer } from "mobx-react";
+import React, { useEffect, useState } from "react";
+
+import { SimulatorContentModelType } from "../model/simulator-content";
+import { ITileProps } from "../../../components/tiles/tile-component";
+import { BasicEditableTileTitle } from "../../../components/tiles/basic-editable-tile-title";
+
+import "./simulator-tile.scss";
+
+export const SimulatorToolComponent: React.FC<ITileProps> = observer((props) => {
```

Comment:
It is better to use the function form here so MST issues are easier to track and you don't need the display name below. Slack thread here: https://concord-consortium.slack.com/archives/C0303QUNU/p1668392500501929